python_modules/dagster/dagster/_utils/schedules.py

```diff
@@ -55,6 +55,23 @@ def cron_string_iterator(
     start_offset: int = 0,
 ) -> Iterator[datetime.datetime]:
     """Generator of datetimes >= start_timestamp for the given cron string."""
+    # leap day special casing
+    if cron_string == "0 0 29 2 *":
```

Comment:
if we do go forward with this, will need to shift and handle other minute/hour values